docs/validator/economics.md

```diff
@@ -0,0 +1,132 @@
+4. The exchange value of NEAR tokens
+5. The opportunity cost of other PoS tokens
+
+**Total of NEAR tokens at stake**
+NEAR rewards algorithm mint 5% new tokens every year. Therefore, the initial supply of 1 billion NEAR will generate 50 million tokens (5% of 1 billion) in stake rewards.
+This reward is fixed, regardless of the stake:
+- A relatively small stake of 100 million (10% of 1 billion) will generate a nominal return of 50% (50 million NEAR for a stake of 100 million), thus attracting more staking.
+- On the other hand, a large stake of 800 million (80% of 1 billion) will return 6.25% in rewards (50 million NEAR for a stake of 800 million), which may become too low to lock so many tokens.
+
+<blockquote class="info">
+    <strong>did you know?</strong><br><br>
+    The protocol automatically re-stakes your rewards at the end of each epoch, so you may expect up to 5% of additional stake every year.
+</blockquote>
+
+**Median of the stake across validators**
+A high median will have fewer validators with large stakes, thus introducing higher barriers of entry for any new `proposal`. On the other hand, a low median decreases the needed stake, introducing more turnaround and competition between validators.
+You may expect the latter, assuming that delegators will spread their stake across multiple validators to hedge risks.
+
+**Protocol fees and inflation**
+In detail, rewards follow the formula `total_supply * 0.05/730` of NEAR tokens at the end of every `epoch` (producing 5% of inflation per year). However, the protocol *burns* the costs of `tx_fees + state_rent` at the end of each block - such that rewards increase `total_supply` and fees reduce it. 
+Specifically, the protocol mints new tokens at the end of each epoch, while burning `tx_fees + state_rent` at each block.
+
+<blockquote class="info">
+    <strong>did you know?</strong><br><br>
+    Beyond a certain level of usage, the protocol will burn more tokens than minting new ones, decreasing NEAR's total supply.
+</blockquote>
+
+**Exchange value of NEAR tokens**
+Validators pay operating expenses in their local currency, and they may have to exchange their NEAR rewards. Most Proof-of-Work miners face similar problems (e.g., energy bills and maintenance personnel), so we can apply the same principles: an appreciating token leaves more operating margin opening opportunities for new validators. On the other hand, a depreciating token may 'squeeze out' the least efficient validators, and centralize the stake.
+Since rewards are fixed, this can be considered a *zero-sum game* influenced by the exchange value of the token.
+
+**Opportunity cost of other PoS tokens**
+Assuming zero barriers to acquire or exchange PoS tokens, every delegator will measure inflation, price fluctuations and potential returns of the same stake across multiple Proof-of-Stake ledgers. Therefore, you have to consider a return threshold, which may impose fee adjustments to stay in the market.
```

Comment:
Hm, I'm not sure this is worth pointing out here.
The assumption is totally wrong, as buying even a $1m of tokens is extremely hard in anything that is not BTC.
Plus if you have delegation, users who delegate may have other motives (like using the network) on top.
My point, I don't think we should have this in our docs

Reply:
It's there, but agree it falls easily out of scope. Removed